api-docs/apiLive.htm

```diff
@@ -13772,6 +13803,111 @@ <h2 class="flybar-button">Template</h2>
 				</table>
 			</div>
 		</div>
+					<a id="workingdays_template" name="workingdays_template"
+				class="old-syle-anchor">&nbsp;</a>
+			<div class="method-section">
+				<div class="method-description">
+					<h4>Working Days Template</h4>
+					<p>This is a convenience resource. It can be useful when
+						building maintenance user interface screens for working days.
+					</p>
+					<p>Example Request:</p>
+					<div class=apiClick>workingdays/template</div>
+				</div>
+				<div class="method-example">
+					<code class="method-declaration">
+GET https://DomainName/api/v1/workingdays/template
+					</code>
+					<code class="method-response">
+{
+    "repaymentRescheduleOptions": [
+        {
+            "id": 1,
+            "code": "RepaymentRescheduleType.same.day",
+            "value": "same day"
+        },
+        {
+            "id": 4,
+            "code": "RepaymentRescheduleType.move.to.previous.working.day",
+            "value": "move to previous working day"
+        },
+        {
+            "id": 3,
+            "code": "RepaymentRescheduleType.move.to.next.repayment.meeting.day",
+            "value": "move to next repayment meeting day"
+        },
+        {
+            "id": 2,
+            "code": "RepaymentRescheduleType.move.to.next.working.day",
+            "value": "move to next working day"
+        }
+    ]
+}
+					</code>
+				</div>
+			</div>
+		<a id="workingdays_list" name="workingdays_list" class="old-syle-anchor">&nbsp;</a>
+		<div class="method-section">
+			<div class="method-description">
+				<h4>List Working days</h4>
+				<p>Example Requests:</p>
+				<div class=apiClick>workingdays</div>
+				<br>
+				<br>
+			</div>
+			<div class="method-example">
+				<code class="method-declaration">
+GET https://DomainName/api/v1/workingdays
+				</code>
+				<code class="method-response">
+[
```

Comment:
Has been converted to a JSON object (instead of array) while merging the pull request